starfish/core/spots/FindSpots/h_max.py

```diff
@@ -11,7 +11,8 @@
 
 from starfish.core.imagestack.imagestack import ImageStack
 from starfish.core.spots.FindSpots import spot_finding_utils
-from starfish.core.types import Axes, Features, SpotAttributes, SpotFindingResults
+from starfish.core.types import Axes, Features, PerImageSpotResults, SpotAttributes, \
```

Comment:
i would wrap this like how you did it for `starfish/core/spots/DecodeSpots/trace_builders.py`